src/wizard/javascript/angular.md

```diff
@@ -23,11 +23,19 @@ You should `init` the Sentry browser SDK as soon as possible during your applica
 import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import * as Sentry from '@sentry/angular';
+import { Integrations } from "@sentry/tracing";
 
 import { AppModule } from './app/app.module';
 
 Sentry.init({
   dsn: '___PUBLIC_DSN___',
+  integrations: [
+    new Integrations.BrowserTracing({
+      tracingOrigins: ["localhost", "https://yourserver.io/api"],
```

Comment:
@HazAT Is there a reason we have `["localhost", "https://yourserver.io/api"]` here instead of `['localhost', /^\//]` this threw me off when I was trying to implement it for the first time (I am not an Angular expert by any means).